### PR TITLE
epi-2865 info on passing 'id' into options for run service

### PR DESCRIPTION
**Pull request by mmrj**

### CODE REVIEW

**Comment by mmrj** [comment]
docs followup for https://issues.forio.com/browse/EPICENTER-2865

note that one place the use of "id" doesn't work / is a weird is in the rs.introspect, which _requires_ either `runID` or `model` and does not work if you pass in only `id` in the options. plus i'm not sure what it would mean to pass in one `runID` and then another `id`.

File: src/service/run-api-service.js [hunk 1]
```diff
@@ -12,6 +12,7 @@
  *
  * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects).
  * * `project`: Epicenter project id.
+ * * `id`: (Optional) The run id of an existing run. If there is a specific run you'd like to work with, you can include it.
```
Comment (narenranjit) [inline src/service/run-api-service.js]: `id` is just an alias for filter (i.e. setting either would do the same thing), think that's worth noting.

Comment (narenranjit) [inline src/service/run-api-service.js]: `id` is just an alias for `filter`, think that's worth calling out (I know it's mentioned in the config section below but i'm assuming this appears above that?)

Comment (mmrj) [inline src/service/run-api-service.js]: yes, it does appear above the config section.
to clarify though -- what would it mean to create a run service with a filter that returns several runs? that e.g. if you called rs.do('myOper') (<-- note no options arg) you'd be calling it on everything in the filter?

Comment (narenranjit) [inline src/service/run-api-service.js]: Yes in theory -- but in practice you'll get a response back saying operations on multiple runs aren't currently supported (assuming that error message made it's way from Lua to Java).

Comment (narenranjit) [inline src/service/run-api-service.js]: It's more useful for read operations (query/ variables().query() etc)

**Comment by narenranjit** [comment]
Re: your comment on introspection - the following _should_ work

```
var rs = new F.service.Run({ .. id: run1 });
rs.introspect(); //introspect run1
rs.instrospect({ runID: run2 }); //introspect run2

```
if it doesn't it's a bug.

**Comment by mmrj** [comment]
re introspection -- yes, that works, thanks

File: src/service/run-api-service.js (hunk 1 above)
Comment (mmrj) [inline src/service/run-api-service.js]: rs.query doesn't respect id though (either passed to the call or passed to the instantiation of Run Service). we can look at examples when we meet shortly?

**Pull request merged**